.vale.ini

```diff
@@ -8,32 +8,38 @@ Vocab = Docs
 [*.{md,mdx}]
 BasedOnStyles = nf-core, Vale
 Vale.Spelling = NO
+Vale.Terms = NO
```

Comment:
are we not loosing a lot of things then with disabling the vale.terms? in all these files, also below